src/shared/components/mutationTable/column/AlleleCountColumnFormatter.tsx

```diff
@@ -48,4 +48,13 @@ export default class AlleleCountColumnFormatter
             </div>
         );
     }
+    
+    public static getReads(sample:Mutation, dataField:string): string
```

Comment:
This isn't well-named, this function could really be summarized as `(sample ? sample[dataField] : '')`. It doesn't have anything to do with Reads